.github/workflows/coverage-ratchet.yml

```
@@ -0,0 +1,243 @@
+          });
+
+    - name: Upload coverage artifacts
+      if: always()
+      uses: actions/upload-artifact@v4
+      with:
+        name: coverage-report
+        path: |
+          cover/
+          coverage_output.txt
+          coverage_report.md
+        retention-days: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)